app/models/asset.rb

```diff
@@ -85,11 +85,15 @@ class Asset
 
     after_transition to: :uploaded do |asset, _|
       asset.save!
-      asset.remove_file!
```

Comment:
As of CarrierWave 1.0.0, calling `remove_file!` method would remove the `file` property from the Asset model; but other code, called later, depends on the `file` property being present even after the file is uploaded to S3.

So I've replaced this with `FileUtils.remove_file(file.path)`, which is functionally equivalent to the pre-upgrade logic.

I've also extracted these two lines to the `delete_file_from_nfs` method below, so that it can be reused in `delete_asset_file_from_nfs_worker.rb`.